boards/arm/thingy53_nrf5340/Kconfig

```diff
@@ -59,6 +59,52 @@ config DOMAIN_CPUNET_BOARD
 	  another board. For example hci_rpmsg on the nRF5340_cpunet for
 	  Bluetooth applications.
 
+config BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+	bool "MCUmgr and OTA DFU"
+	select MCUMGR
+	select MCUMGR_SMP_BT
+	select MCUMGR_CMD_IMG_MGMT
+	select MCUMGR_CMD_OS_MGMT
+	select MCUMGR_GRP_ZEPHYR_BASIC
+	select MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE
+	select OS_MGMT_MCUMGR_PARAMS
+	select NORDIC_QSPI_NOR
+	select MCUMGR_SMP_REASSEMBLY_BT
+	help
+	  Enable this option to include MCUmgr in the build which will enable
+	  the Bluetooth transport and image management group, which allows
+	  for firmware over the air update support of the device.
+
+config MCUMGR_SMP_BT_AUTHEN
+	default n if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+
+config UPDATEABLE_IMAGE_NUMBER
+	default 2 if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+
+config BT_L2CAP_TX_MTU
+	default 498 if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+
+config BT_BUF_ACL_RX_SIZE
+	default 502 if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+
+config BT_BUF_ACL_TX_SIZE
+	default 502 if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
+
+config BT_CTLR_DATA_LENGTH_MAX
+	default 251 if BOARD_ENABLE_MCUMGR_BT_OTA_DFU
```

Comment:
We need to match this configuration in the Network Core image

Reply:
See the child image configuration in the SMP Server sample:

https://github.com/nrfconnect/sdk-zephyr/blob/main/samples/subsys/mgmt/mcumgr/smp_svr/child_image/hci_rpmsg.conf

Reply:
As above:

> You can't know if this option has been selected from the network core image so have no way to know

So either (for the network core) these are always left at their defaults or set to their maximums whenever bluetooth is enabled for the network core - which do we want to go with?

Reply:
Fixed to maximum by default